src/Core/Services/Implementations/CipherService.cs

```diff
@@ -118,6 +121,15 @@ public class CipherService : ICipherService
                 }
                 else
                 {
+                    // Make sure the user can save new ciphers to their personal vault
+                    var userPolicies = await _policyRepository.GetManyByUserIdAsync(savingUserId);
+                    if (userPolicies != null)
+                    {
+                        if (userPolicies.Any(policy => policy.Type == PolicyType.PersonalOwnership && policy.Enabled))
```

Comment:
can be combined into one condition

Reply:
This comment made me realize that there was another layer of enforcement conditionals missing to properly apply the policy.  I've updated this PR and added the necessary changes.